### PR TITLE
updates to headers for ToCs

### PR DESCRIPTION
**Pull request by TownLake**
* Fixes to header sizing, header links for consistency and table of contents

### CODE REVIEW

File: products/access/src/content/access-service-auth/mutual-tls-authentication.md [hunk 1]
```diff
@@ -70,158 +66,11 @@ curl -sv https://auth.example.com --cert example.pem --key key.pem
...
-You can use the Cloudflare PKI toolkit to generate a certificate revocation list (CRL), as well. This list will contain client certificates that are revoked.
-1. Get the serial number from the client certificate generated earlier. Add that serial number, or any others you intend to revoke, in hex format in a text file. This example uses a file named `serials.txt`.
-
-2. Create the CRL with the following command.
-```bash
-cfssl gencrl serials.txt ../mtls-test/ca.pem ../mtls-test/ca-key.pem | base64 -D > ca.crl
-You will need to add this to your server or enforce the revocation in a Cloudflare Worker.
-```
-
-# Mutual TLS Testing
```
Comment (adamschwartz) [inline products/access/src/content/access-service-auth/mutual-tls-authentication.md:220]: Did you mean to remove this whole block?

**Review by adamschwartz — CHANGES_REQUESTED** [review]
Thank you for doing this. There’s one small bit I wasn’t sure about (see below). Otherwise looks good to me.

File: products/access/src/content/access-service-auth/mutual-tls-authentication.md (hunk 1 above)
Comment (TownLake) [reply products/access/src/content/access-service-auth/mutual-tls-authentication.md:220]: It is a copy-paste issue - a duplicate of the content above it.

Comment (adamschwartz) [reply products/access/src/content/access-service-auth/mutual-tls-authentication.md:220]: @TownLake let me know it’s duplicate content.

**Review by adamschwartz — APPROVED** [review]

**Pull request merged**